flowcraft/generator/components/assembly.py

```diff
@@ -177,3 +177,39 @@ def __init__(self, **kwargs):
                     "Use with caution."
             }
         }
+
+class Abyss(Process):
+    """ABySS process template interface
+
+    This process is set with:
+
+        - ``input_type``: fastq
+        - ``output_type``: assembly
+        - ``ptype``: assembly
+
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = "fasta"
+
+        # self.dependencies = ["integrity_coverage"]
```

Comment:
Do you need this line? If you don't need it, remove it.

Reply:
I'll remove it!